include/aspect/assembly.h

```diff
@@ -104,6 +104,7 @@ namespace aspect
           std::vector<SymmetricTensor<2,dim> > grads_phi_u;
           std::vector<double>                  div_phi_u;
           std::vector<Tensor<1,dim> >          velocity_values;
+          std::vector<double>                  velocity_divergences;
```

Comment:
so I'm still wondering if we need this field, or if we should just compute it as `trace(grads_phi_u)` whenever we need it. I generally don't particularly like storing redundant information because it tends to diverge